test/plugins/scenarios/gozero/plugin.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+entry-service: http://${HTTP_HOST}:${HTTP_PORT}/user/info
```

Comment:
I can't see the excepted file to validate all generated tracing data?

Reply:
The file path is: `test/plugins/scenarios/gozero/handler.go`. I compressed the application code generated by the goZero tool goctrl and consolidated the `HTTP` and `RPC` code into one directory for easier testing.

Reply:
@zjh129 I think the question is, where is the validation? You have the sample application, and with this booting script. Then the CI tool requires the `excepted.yml`.

An reference, https://github.com/apache/skywalking-go/blob/main/test/plugins/scenarios/dubbo/config/excepted.yml

Otherwise, CI can't tell whether your codes are able to work. 

We as maintainers can't validate all through manual and logic setup, because it is not practical and costs too much time.

Reply:
I mean you have to submit the tracing data which generated from the skywalking-go. Such as when we enhance the `net/http` framework, we have to generate [an excepted.yaml file](https://github.com/apache/skywalking-go/blob/main/test/plugins/scenarios/http/config/excepted.yml) to validate the tracing data are reported success or not.